templates/include/_navbar.html.twig

```diff
@@ -3,67 +3,65 @@
-                            <a class="nav-link text-light" href="{{ path('contact_show') }}">Contact</a>
-                        </li>
+    <nav class="navbar navbar-expand-xl  navbar-dark bg-info fixed-top">
+        <div class="container-fluid ">
+            <a class="navbar-brand img d-none d-lg-block text-light" href="{{ path('home') }}">
+                <img class="logo mr-2" src="{{ asset('build/images/LOGO-Blanc.png') }}"
+                     alt="Logo"/>Damien Gouveia Coach Sportif</a>
+            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavDropdown"
+                    aria-controls="navbarNavDropdown" aria-expanded="false" aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon"></span>
+            </button>
+            <div class="collapse navbar-collapse justify-content-end text-light" id="navbarNavDropdown">
+                <ul class="navbar-nav">
+                    <li class="nav-item dropdown mx-3">
+                        <a class="nav-link dropdown-toggle text-light" href="#" id="navbarDropdownMenuLink" role="button"
+                           data-toggle="dropdown" aria-expanded="false">
+                            Les services
+                        </a>
+                        <ul class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink">
+                            <li><a class="dropdown-item" href="{{ path('activity_index') }}">Activité adaptée</a></li>
+                            <li><a class="dropdown-item" href="{{ path('fitness') }}">Remise en forme</a></li>
+                            <li><a class="dropdown-item" href="{{ path('walking_index') }}">Marche nordique</a></li>
+                            <li><a class="dropdown-item" href="{{ path('company_index') }}">Santé en entreprise</a></li>
+                        </ul>
+                    </li>
+                    <li class="nav-item mx-3">
+                        <a class="nav-link text-light" href="{{ path('gym') }}">La salle d’entrainement</a>
+                    </li>
+                    <li class="nav-item mx-3">
+                        <a class="nav-link text-light" href="{{ path('who') }}">Qui suis-je ?</a>
+                    </li>
+                    <li class="nav-item mx-3">
+                        <a class="nav-link text-light" href="{{ path('tarifs_index') }}">Tarifs</a>
+                    </li>
+                    <li class="nav-item mx-3">
+                        <a class="nav-link text-light" href="{{ path('media_index') }}">Média</a>
+                    </li>
+                    <li class="nav-item mx-3">
+                        <a class="nav-link text-light" href="{{ path('contact_show') }}">Contact</a>
+                    </li>
+                    {% if app.user %}
+                    <li class="nav-item mx-3">
+                        <a class="nav-link text-light" href="{{ path('profile_user', {email: app.user}) }}">
```

Comment:
je comprend pas le param email:app.user qui semble non utilisé dans la route profile_user ? 
et dans tous les cas inutile à passer en param puisque app.user est récupérable dans tous les fichiers Twig

Reply:
de plus, toutes vos routes/securisation sur la partie profile est à retravailler car si je tape http://localhost:8000/compte/2/edit j'y accède depuis l'user 1 !!!
ne passez pas d'id en paramètres des routes mais pour travailler sur un user, recupérez le soit via app.user dans le twig, soit via $this->getUser() si besoin dans le controller